### PR TITLE
Use replacing placeholder in exports

### PR DESCRIPTION
**Pull request by spiess-demos**
**Ticket:** https://yaits.demos-deutschland.de/T32104

We also want to replace `{year}` in map attributions with the current year in exports.

Besides creating a little service function for the replacement, the misleading trans key `copyright.sh` is renamed here.

Related Project PR: https://github.com/demos-europe/demosplan-project-diplanbau/pull/118

### Todo

The PDF export shows "...year", the replacement seems to not work there. This must be fixed.

### How to review/test
In all exports, the string `{year}` should be replaced by the current year in map attributions.

### PR Checklist

- [ ] Tests updated/created
- [ ] Update documentation
- [X] Link all relevant tickets
- [X] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Logic/AssessmentTable/AssessmentTableServiceOutput.php [hunk 1]
```diff
@@ -440,7 +440,7 @@ public function buildOriginalStatementDocxExport(Procedure $procedure, array $pr
                 if (null !== $image) {
                     $docxImageTag = $this->getDocxImageTag($image);
                     Html::addHtml($section, $docxImageTag);
-                    $section->addText($this->translator->trans('copyright.sh'));
+                    $section->addText($this->translator->trans('map.attribution.exports', ['year' => date('Y')]));
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Logic/AssessmentTable/AssessmentTableServiceOutput.php:443]:
```suggestion
                    $section->addText($this->translator->trans('map.attribution.exports', ['currentYear' => date('Y')]));
```

File: demosplan/DemosPlanCoreBundle/Utilities/Map/MapScreenshotter.php [hunk 2]
```diff
@@ -196,8 +204,7 @@ public function makeScreenshot(
                 return $this->makeScreenshotWmts($polygon, $copyrightText);
             } else {
                 if (null === $copyrightText) {
-                    $copyrightText = '© basemap.de BKG (www.basemap.de) /'.
-                    'LVermGeo SH (www.LVermGeoSH.schleswig-holstein.de)';
+                    $copyrightText = $this->translator->trans('map.attribution.exports', ['year' => date('Y')]);
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Utilities/Map/MapScreenshotter.php:207]:
```suggestion
                    $copyrightText = $this->translator->trans('map.attribution.exports', ['currentYear' => date('Y')]);
```

File: demosplan/DemosPlanCoreBundle/Utilities/Map/MapScreenshotter.php [hunk 3]
```diff
@@ -264,7 +271,7 @@ public function makeScreenshotWmts(
     ): ?string {
         try {
             $copyrightText = $copyrightText
-                ?? '© basemap.de BKG (www.basemap.de) / LVermGeo SH (www.LVermGeoSH.schleswig-holstein.de)';
+                ?? $this->translator->trans('map.attribution.exports', ['year' => date('Y')]);
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Utilities/Map/MapScreenshotter.php:274]:
```suggestion
                ?? $this->translator->trans('map.attribution.exports', ['currentYear' => date('Y')]);
```

**Review by breierDemos — APPROVED** [review]

**Comment by breierDemos** [comment]
PR to reduce code duplication: https://github.com/demos-europe/demosplan-core/pull/1481, raised by sonarcloud

**Pull request merged**